src/components/typography/typography.spec.tsx

```diff
@@ -588,7 +597,7 @@ describe("Typography", () => {
 });
 
 describe("List", () => {
-  it.each([
+  it.each<[keyof JSX.IntrinsicElements, string]>([
```

Comment:
I think you could also do something like this:

```
  ["ul", "square"] as const,
  ["ol", "decimal"] as const
```

Reply:
I think the `as const` assertion on the whole array of the two 2-tuples is probably the best way to go here?